drake/util/Polynomial.cpp

```diff
@@ -420,57 +421,59 @@ bool Polynomial<CoefficientType>::isApprox(const Polynomial& other,
   return getCoefficients().isApprox(other.getCoefficients(), tol);
 }
 
-const string name_chars = "@#_.abcdefghijklmnopqrstuvwxyz";
-const unsigned int num_name_chars = 30;  // length of the string above
-const unsigned int name_length = 4;
-const unsigned int max_name_part = 923521;  // (num_name_chars+1)^name_length;
+const char kNameChars[] = "@#_.abcdefghijklmnopqrstuvwxyz";
+const unsigned int kNumNameChars = sizeof(kNameChars);
```

Comment:
I believe a `- 1` is missing here, to account for the string terminator in sizeof?